kork-web/src/main/java/com/netflix/spinnaker/config/okhttp3/OkHttpClientProvider.java

```diff
@@ -45,8 +49,24 @@ public OkHttpClient getClient(ServiceEndpoint service) {
     return getClient(service, List.of());
   }
 
+  public OkHttpClient getClient(ServiceEndpoint service, boolean skipEncodeCorrection) {
+    return getClient(service, List.of(), skipEncodeCorrection);
+  }
+
   public OkHttpClient getClient(ServiceEndpoint service, List<Interceptor> interceptors) {
+    return getClient(service, interceptors, false);
```

Comment:
```suggestion
    return getClient(service, interceptors, false /* skipEncodeCorrection */);
```
and please add a comment about why we're including encode correction by default.  It seems obvious at the moment, but our future selves may forget.